Lib/fontPens/penTools.py

```diff
@@ -98,28 +98,51 @@ def getQuadraticPoint(t, pt0, pt1, pt2):
     return x, y
 
 
+def getCubicPointPath(increments, pt0, pt1, pt2, pt3):
```

Comment:
Nice! I would call it `getCubicPoints()`, though, and the `increments` argument `ts`. This may be a little more consistent with `bezierTools.py`. Actually I would prefer the argument order to be `getCubicPoints(pt0, pt1, pt2, pt3, ts)` but that is inconsisten with `getCubicPoint()` so never mind that...

Reply:
That makes sense. Done.